container/root/etc/services-available/php-fpm/finish

```diff
@@ -4,4 +4,7 @@
 if { s6-test ${1} -ne 0 }
 if { s6-test ${1} -ne 256 }
 
-s6-svscanctl -t /var/run/s6/services
+# Suppress superfluous time-of-check-time-of-use (TOCTOU) message
```

Comment:
is `time-of-check-time-of-use` or is it supposed to be just `check-time-of-use`?

Reply:
its specifically TOCTOU